includes/wikia/GlobalFunctions.php

```diff
@@ -916,19 +916,19 @@ function wfTimeFormatAgo( $stamp, $hideCurrentYear = true ) {
 		// format is date, with a year (July 26, 2008)
 		$res = $wgLang->date( wfTimestamp( TS_MW, $stamp ) );
 	} elseif ( $ago < 60 ) {
-		// Under 1 min: to the second (ex: 30 seconds ago)
+		// Under 1 min: to the second (ex: 30s)
 		$res = wfMsgExt( 'wikia-seconds-ago', array( 'parsemag' ), $ago );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this legacy wfMsg* function call with one of the new i18n functions. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=wikia-php-rules%3ALegacyWfMsgUsage)